config/routes.rb

```diff
@@ -1,5 +1,9 @@
 Rails.application.routes.draw do
   devise_for :users
+
+  resources :categories do
+    resources :expenses
+  end
```

Comment:
- [ ] Please, create only routes that are needed for this application. This is all about clarity, tidying up, and protecting against unexpected errors or security holes.

In the default, unrestricted, case all seven default routes are created. If a user calls a route that is defined but not used, Rails will attempt to call the relevant controller action, even if it doesn’t exist. This can lead to errors or, worse, unexpected behaviour. Please use the `only` and `except` resources options to create only the routes that are needed. 👍 

#### Hint - This is an example of how to do such:

```ruby
resources :orders, except: %w[destroy] do
  resources :products, only: %w[show]
end
```